Letterbook.Web.Tests.E2E/README.md

```diff
@@ -0,0 +1,32 @@
+﻿# E2E Testing for Letterbook.Web
+
+> Note: this project is using **NUnit** not **XUnit** like other test projects we use.
+
+## Setup
+
+This test package uses [Playwright](https://playwright.dev). While most dependencies are provided automatically for you, Playwright will require you to install test browsers to run these tests.
+
+You can find out more about this at https://playwright.dev/dotnet/docs/intro
+
+Or use
+
+```shell
+dotnet build
+pwsh bin/Debug/netX/playwright.ps1 install
+```
```

Comment:
I love the readme, thank you! Are you able to add similar instructions for linux and/or mac? If not, that's ok, we can do it later.